app/views/programs/show.html.haml

```diff
@@ -0,0 +1,45 @@
+%div
+  %h3
+    Program
+  %p
+    The largest official gathering of the year, RailsConf brings together top talent, companies, and project representatives from around the world. Learn and build with the best in sessions, workshops, keynotes and parties.
```

Comment:
"RailsConf" can't be hardcoded...

Reply:
Got it. Although we might want to change this we do some reconfiguring of the event object.